.github/workflows/hermetic.yml

```diff
@@ -36,8 +40,25 @@ jobs:
           echo ${BUILD_TAG}
           echo "build_tag=${BUILD_TAG}" >> $GITHUB_OUTPUT
           cat $GITHUB_OUTPUT
+  publish-suite:
+    runs-on: ubuntu-latest
+    needs: generate-matrix
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to Public ECR
+      uses: docker/login-action@v2
+      with:
+        registry: public.ecr.aws
+        username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      env:
+        AWS_REGION: us-east-1
+    - name: Build and Publish
+      run: make BUILD_TAG=${{ needs.generate-matrix.outputs.build_tag }} publish-suite
   build-driver:
-    name: Build 'ceramic-tests-driver' binary
+    name: Build 'ceramic-tests-hermetic-driver' binary
```

Comment:
Should we shorten to `hermetic-test-driver` and `durable-test-driver` (or `persistent-test-driver`)?